src/Symfony/Component/Notifier/Exception/UnsupportedSchemeException.php

```diff
@@ -140,6 +140,10 @@ class UnsupportedSchemeException extends LogicException
             'class' => Bridge\OvhCloud\OvhCloudTransportFactory::class,
             'package' => 'symfony/ovh-cloud-notifier',
         ],
+        'ring-central' => [
```

Comment:
```suggestion
        'ringcentral' => [
```

sorry, this is the scheme and it should not contain a dash, my fault